src/test/rustdoc-gui/mobile.goml

```diff
@@ -0,0 +1,9 @@
+goto: file://|DOC_PATH|/test_docs/index.html
+size: (600, 600)
+
+// The out-of-band info (source, stable version, collapse) should be below the
+// h1 when the screen gets narrow enough.
+assert-css: (".main-heading", {
+  "display": "flex",
+  "flex-direction": "column"
+})
```

Comment:
There is a change in text too. Can you check that the since only appears on mobile view maybe please?

Reply:
Added! I rebased on top of #92830 so the staged_api test would be available.

Reply:
Thanks!